src/commands/push-image.yml

```diff
@@ -14,11 +14,16 @@ parameters:
     description: Name of an Amazon ECR repository
 
   tag:
-    description: A docker image tag (default = latest)
+    description: Optionally define a single docker image tag to push, instead of pushing all the repo's tags (default = "")
     type: string
-    default: "latest"
+    default: ""
```

Comment:
I think it'd be good to keep the original default to avoid unexpected behaviour for some users